### DISCUSSION TITLE
Add a Skip-The-Line Queue feature

### DISCUSSION THREAD

**Discussion by MemesRCT** [OpenRCT2/OpenRCT2#17570] [Feature requests]
It would be cool to see park guests pay to bypass the standard queue line for any park's popular rides/attractions like in real parks. Please add this feature to OpenRCT2!

**Comment by astraylife1** [comment]
I've tried to implement this with loading and unloading stations. I simply add an entrance to the exit station as well and make a shorter queue for exiting riders. The issue is that all the guests try to get to the shorter queue and complain if they can't find it.
